configure.ac

```diff
@@ -4238,8 +4235,13 @@ fi
 
 AC_CHECK_FUNCS(getnameinfo)
 
+dnl backwards compatibility: autoconf 2.71 deprecates AC_HEADER_TIME
+dnl TIME_WITH_SYS_TIME works on all supported systems that have sys/time.h
+AS_VAR_IF([ac_cv_header_sys_time_h], [yes], [
+  AC_DEFINE([TIME_WITH_SYS_TIME], 1, [Define to 1 if you can safely include both <sys/time.h> and <time.h>.])
```

Comment:
Why do we need to define this? It is no longer used in the code base.

Reply:
3rd party software may depend on the macro. There are a ton of C extensions in the world. I'm hesitant to remove feature macros.

Reply:
Doh, of course. Thanks.